testing/correctness/tests/autoscale_grow/autoscale_grow.py

```diff
@@ -88,7 +89,7 @@ def _test_autoscale_grow(command, worker_count=1):
     sources = 1
     workers = 1
     joiners = worker_count - workers
-    res_dir = '/tmp/res-data'
+    res_dir = '/tmp/res-data.%f' % time.time()
```

Comment:
Is it helpful for what we're doing to append the time to the directory name? Would it make more sense to use something like the `tempfile` module?

Reply:
Heh, I didn't know that `tempfile` did directories also.

Wall clock time was a convenient source of randomness and very nearly unique (well, without episodes of system clock skew).  I wouldn't recommend this method for production use, where an attacker guessing the dir's name could do bad things.  I suppose that we could worry about attacks during Wallaroo testing.  Or worry about killing a kitten each time someone doesn't use an allegedly good library function that allegedly avoids such attacks.

Opinions?

Reply:
Also:

```python
>>> import tempfile, time
>>> tempfile.mktemp(dir="/tmp/yoo")
'/tmp/yoo/tmprm296O'
>>> tempfile.mktemp(dir="/tmp/yoo", prefix="%f|"%time.time(), suffix="|%f"%time.time())
'/tmp/yoo/1517616923.463343|do9Qsc|1517616923.463348'
```